README.md

```diff
@@ -196,7 +196,7 @@ Give a ⭐️ if you like this project!
 
 - **How to use the project**
 
-  - ```git clone https://github.com/juaryimami/math-magicians```
+  - ```git clone https://github.com/juaryimami/metrics-webapp```
 
 - **How to contribute to the project**
 
```

Comment:
- [ ]  In your Readme file, in the Acknowledgements part, you should give appropriate credit to Nelson Sakwa the author of the original design, as required by the Creative Commons License.

Reply:
ok I will do it soon. Thank you so much sir!!!